scripts/zones/West_Ronfaure/npcs/Doladepaiton_RK.lua

```diff
@@ -9,27 +9,80 @@ package.loaded["scripts/zones/West_Ronfaure/TextIDs"] = nil;
 
 require("scripts/globals/conquest");
 require("scripts/zones/West_Ronfaure/TextIDs");
+require("scripts/globals/garrison");
 
+-- Conquest
 local guardnation = NATION_SANDORIA; -- SANDORIA, BASTOK, WINDURST, 4 = jeuno
 local guardtype   = 3;        -- 1: city, 2: foreign, 3: outpost, 4: border
 local region      = RONFAURE;
 local csid        = 0x7ffb;
 
+-- Garrison
+local garrison_csid = 32753
+
+-----------------------------------
+-- onSpawn
+-----------------------------------
+function onSpawn(npc)
+    npc:setLocalVar("GarrisonState", IDLE_GARRISON)
+end
+
 -----------------------------------
 -- onTrade Action
 -----------------------------------
 
 function onTrade(player,npc,trade)
```

Comment:
You might want to comment out the print statements to avoid filling the console with output or alternatively have method/variable in garrison.lua that defines a logging method and the server operator can toggle that value to debug garrison.

```lua
--garrison.lua

LOG_GARRISON = false

function garrisonPrint(msg)
  if LOG_GARRISON then
    print(msg)
  end
end
```

Reply:
The print statement were only temporary, mainly pushed this up for scrutiny on the framework, but I will make sure they are all gone by the time we get it prepared for merging.